subsys/logging/Kconfig

```diff
@@ -44,6 +44,19 @@ config LOG_DEFAULT_LEVEL
 	  - 3 INFO, default to write LOG_LEVEL_INFO
 	  - 4 DEBUG, default to write LOG_LEVEL_DBG
 
+config LOG_OUTPUT
```

Comment:
How is that used? Is it some macro magic? Can't find any usage of it except the one unused macro.